guides/source/action_controller_overview.md

```diff
@@ -110,7 +110,10 @@ class ClientsController < ApplicationController
 end
 ```
 
+NOTE: `params` hash is not a plain old Hash; instead, it is [`ActionController::Parameters`][] object. While it behaves like Hash, it does not inherit from Hash class.
```

Comment:
In the getting started guide we capitalize hash in "`params` Hash".

```suggestion
NOTE: The `params` Hash is not a plain old Ruby Hash; instead, it is an [`ActionController::Parameters`][] object. While it behaves like Hash, it does not inherit from Hash.
```

Reply:
There are five other examples of "`params` hash" in this guide already, though.

Reply:
Ah, missed those while grepping. It seems "`params` hash" is more common.

Reply:
@JunichiIto Could you add the missing articles (`The` and `an`)?

Reply:
@p8 I added your suggestion to batch. Is it okay?

Reply:
Thanks @JunichiIto !
Can you squash your commits?

Reply:
@p8 Sure. I squashed and pushed my commit.